### PR TITLE
R report

### PR DESCRIPTION
**Pull request by ishantripathi**
Added functionality of generating report for only the projects that are required:

**How to do it?**
Set the variable in code as per below:-

**->To generate reports for all the projects in institution**
_projects_in_rpt = "all"_

**->if only certain number of projects needs to be in the report, enter the name of projects in the same format as project 1 and project 2 is written.** 

_projects_in_rpt ='"project_name"
"project 1"
"project 2"'_

### CODE REVIEW

**Review by anju-james — DISMISSED** [review]

File: r_report/script/moc.rmd [hunk 1]
```diff
@@ -33,16 +35,78 @@ institution2project_path = paste0(csv_path,"institution2project/",csv_file_name)
 
 
 \vspace{48pt}
+>>>>>>> c3b2f43df380239ea77a7c8c9b6dd8ebc4098105
```
Comment (tsachleben) [inline r_report/script/moc.rmd]: Please Clean up Merge Tags

File: r_report/script/moc.rmd [hunk 2]
```diff
@@ -69,25 +133,40 @@ item_usage <- select(item_ts_minmax,item_id,max_ts,min_ts) %>% group_by(item_id)
 
 
 
+>>>>>>> c3b2f43df380239ea77a7c8c9b6dd8ebc4098105
```
Comment (tsachleben) [inline r_report/script/moc.rmd]: Please clean up merge tags

File: r_report/script/moc.rmd [hunk 3]
```diff
@@ -69,25 +133,40 @@ item_usage <- select(item_ts_minmax,item_id,max_ts,min_ts) %>% group_by(item_id)
...
+active_proj_in_inst <- final_table %>% filter(institution_name == institution_n) %>% select(project_name)
+active_proj_in_inst <- unique(active_proj_in_inst)
+} else {
+  active_proj_in_inst <- read.csv(text = projects_in_rpt)
+}
+=======
 
 active_proj_in_inst <- final_table %>% filter(institution_name == institution_n) %>% select(project_name)
 active_proj_in_inst <- unique(active_proj_in_inst)
+>>>>>>> c3b2f43df380239ea77a7c8c9b6dd8ebc4098105
```
Comment (tsachleben) [inline r_report/script/moc.rmd]: Please clean up merge tags

File: r_report/script/moc.rmd [hunk 4]
```diff
@@ -69,25 +133,40 @@ item_usage <- select(item_ts_minmax,item_id,max_ts,min_ts) %>% group_by(item_id)
...
+<<<<<<< HEAD
+  cat("\n\n\\pagebreak\n")
+  
+=======
 
 
   cat("\n\n\\pagebreak\n")
   
 
+>>>>>>> c3b2f43df380239ea77a7c8c9b6dd8ebc4098105
```
Comment (tsachleben) [inline r_report/script/moc.rmd]: Please clean up merge tags

**Review by tsachleben — CHANGES_REQUESTED** [review]

**Review by tsachleben — APPROVED** [review]

**Review by anju-james — APPROVED** [review]

**Pull request merged**